resources/definitions/tizyx_evy_dual.def.json

```diff
@@ -0,0 +1,60 @@
+
+
+        "machine_extruder_trains":
+        {
+            "0": "tizyx_evy_dual_extruder_0",
+            "1": "tizyx_evy_dual_extruder_1"
+        },
+        "platform": "tizyx_k25_platform.stl",
+        "platform_offset": [0, -4, 0],
+        "first_start_actions": ["MachineSettingsAction"],
+        "supported_actions": ["MachineSettingsAction"]
+    },
+
+    "overrides": {
+        "machine_extruder_count": { "default_value": 2 },
+        "machine_heated_bed": { "default_value": true },
+        "machine_center_is_zero": { "default_value": false },
+        "gantry_height": { "default_value": 500 },
+        "machine_height": { "default_value": 255 },
+        "machine_depth": { "default_value": 255 },
+        "machine_width": { "default_value": 255 },
+        "machine_head_with_fans_polygon": {
+            "default_value": [
+                [25, 49],
+                [25, -49],
+                [-25, -49],
+                [25, 49]
+            ]
+        },
+        "machine_start_gcode":
+        {
+            "default_value": "M82\nG90\nG28 X\nG28 Y\nG28 Z\nG29\nG91\nG1 Z0\nG90\nM82\nG92 E0\nG1 X125 Y245 F3000\nG1 Z0"
+        },
+        "machine_end_gcode":
+        {
+            "default_value": "M104 S0\nM140 S0\nG91\nG1 E-5 F300\nG1 Z+3 F3000\nG1 Y245 F3000\nM84"
+        },
+        "layer_height": { "maximum_value": "(0.8 * min(extruderValues('machine_nozzle_size')))" },
+        "layer_height_0": { "maximum_value": "(0.8 * min(extruderValues('machine_nozzle_size')))" }
```

Comment:
Typically Cura doesn't actually prevent slicing if the result would just be a bad quality print, but only if it's physically impossible (like speed < 0). So please change these to `maximum_value_warning` instead, so that the user gets a warning that the quality might not be good but is allowed to slice.